offchain/authority-claimer/src/checker.rs

```diff
@@ -88,7 +88,7 @@ impl DuplicateChecker for DefaultDuplicateChecker {
         rollups_claim: &RollupsClaim,
     ) -> Result<bool, Self::Error> {
         let proof_context =
-            U256([0, 0, 0, rollups_claim.epoch_index]).encode().into();
```

Comment:
Is this change intended? If yes, please add it in another commit.

Reply:
It's intentional. The way it was before, it was building the proof hex the other way around, e.g. 0x01000000 instead of 0x00000001. The claim contracts were not happy with this, as both the get couldn't find it and submit thought we skipped epochs.